t/ui/18-tests-details.t

```diff
@@ -545,5 +545,25 @@ subtest 'additional investigation notes provided on new failed' => sub {
         'Investigation status content shown');
 };
 
+subtest 'show modules spent time' => sub {
+    $driver->get("/tests/99937");
+    my $tds                = $driver->find_elements(".component");
+    my %modules_spent_time = (
+        zypper_in => '243s ',
```

Comment:
It would look so much nicer with a space between `s` and the number. (`s` stands for seconds and if you would write it out you would also add a space.)